tmplctlr/metrics.go

```diff
@@ -0,0 +1,16 @@
+package tmplctlr
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	managedReleases = prometheus.NewGauge(prometheus.GaugeOpts{
+		Help: "The number of releases managed by this operator",
+		Name: "managed_releases_total",
```

Comment:
I'm not sure on the name of this metric, I picked releases because deployments already has a more specific meaning.  I'm open to suggestions on  better terminology for what we call the bundles of configs managed by lostromos.